### DISCUSSION TITLE
Request for unlocking issue #8015

### DISCUSSION THREAD

**Discussion by MasterInQuestion** [curl/curl#9175] [General]
&#xA0;&#xA0;&#xA0;&#xA0;[ Quote bagder @ CE 2022-07-16 15:24:43 UTC:
[https://github<i>.</i>com/curl/curl/issues/8015#issuecomment-1186218199](https://github.com/curl/curl/issues/8015#issuecomment-1186218199)
&#xA0;&#xA0;&#xA0;&#xA0;I don't recognize that as a (past) problem. Also this is a closed issue about something else. ]
<^>&#xA0;&#xA0;&#xA0;&#xA0;I realized the problem and I need to update my previous comment.
&#xA0;&#xA0;&#xA0;&#xA0;Please unlock the conversation.

**Comment by bagder** [comment]
No. If you have an issue with the current curl version, file a new issue describing the problem. If you suffer from an old problem in an old version, we don't need to hear about it.

**Reply by MasterInQuestion** [reply]
&#xA0;&#xA0;&#xA0;&#xA0;The problem is the post [ [https://github<i>.</i>com/curl/curl/issues/8015#issuecomment-1186088145](https://github.com/curl/curl/issues/8015#issuecomment-1186088145) ] is currently misleading and needs to be edited.

**Reply by bagder** [reply]
No it doesn't. Nobody cares about that. Move on.

**Reply by MasterInQuestion** [reply]
&#xA0;&#xA0;&#xA0;&#xA0;Such assertion may be too imprudent.
&#xA0;&#xA0;&#xA0;&#xA0;.
&#xA0;&#xA0;&#xA0;&#xA0;Nevertheless, would there be any real negative consequence for simply unlocking it?
&#xA0;&#xA0;&#xA0;&#xA0;While the negative consequences of keeping which locked appear to be more significant: possibly causing pointless spawns of the identical issue.

**Reply by bagder** [reply]
Yes, as you seem to be "polluting" a whole range of issues and pull-request with follow-up comments and off-topic debates that we do not want there. Closed issues and pull-requests are done, handled, past-tense. If you have an existing unhandled issue, then file a new issue. If you want to discuss something, do it in a discussion here or even better on a mailing list. The issues and pull-requests are meant to be single topics and when they're closed they're done.

**Reply by MasterInQuestion** [reply]
&#xA0;&#xA0;&#xA0;&#xA0;Would having multiple similar issues all listed independently scattering all around really provide anyhow better experience?

&#xA0;&#xA0;&#xA0;&#xA0;Perfectly determining the relativity of things can be hard or impossible:
&#xA0;&#xA0;&#xA0;&#xA0;Most of my posts that contain "Off-Topic" are actually not directly off-topic: the sections are partly related with the topic but not related enough (to be considered on-topic).
&#xA0;&#xA0;&#xA0;&#xA0;While outright removing them would make the post incomplete.
&#xA0;&#xA0;&#xA0;&#xA0;.
&#xA0;&#xA0;&#xA0;&#xA0;Only 1 example in my posting had been (much) entirely off-topic, but seemingly harmless: [ [https://github<i>.</i>com/curl/curl/issues/9162#issuecomment-1188472508](https://github.com/curl/curl/issues/9162#issuecomment-1188472508) ]. (would commenting in such closed issue really bother?)
&#xA0;&#xA0;&#xA0;&#xA0;"done, handled, past-tense", you may as well just ignore.

&#xA0;&#xA0;&#xA0;&#xA0;Inconsiderately locking threads only impedes the project's development:
&#xA0;&#xA0;&#xA0;&#xA0;|&#x2A;| [https://github<i>.</i>com/curl/curl/issues/9160#event-7024330742](https://github.com/curl/curl/issues/9160#event-7024330742)
&#xA0;&#xA0;&#xA0;&#xA0;|&#x2A;| [https://github<i>.</i>com/curl/curl/issues/9153#event-7024199593](https://github.com/curl/curl/issues/9153#event-7024199593)
&#xA0;&#xA0;&#xA0;&#xA0;.
&#xA0;&#xA0;&#xA0;&#xA0;May your Excellency caution.
\
\
=== Off-Topic ===

&#xA0;&#xA0;&#xA0;&#xA0;Who is in the position to imperate: the way intended: how any particular piece of information to be interpreted?
&#xA0;&#xA0;&#xA0;&#xA0;.
&#xA0;&#xA0;&#xA0;&#xA0;Data, are no more than data. It is all upon: the interpretation.

**Reply by MasterInQuestion** [reply]
https://pastebin.com/raw/YTi5UzLq

**Reply by bagder** [reply]
I don't understand what you're saying here.

**Reply by MasterInQuestion** [reply]
&#xA0;&#xA0;&#xA0;&#xA0;No matter. The pastebin link is the intended edition of the aforementioned post.
&#xA0;&#xA0;&#xA0;&#xA0;Which tagged the incorrect information and made further explanations to the situation.
&#xA0;&#xA0;&#xA0;&#xA0;.
&#xA0;&#xA0;&#xA0;&#xA0;It also partly answered the question in the main post.
